hack/install.sh

```diff
@@ -67,6 +67,9 @@ install_arkade(){
   arkade --help
 }
 
+# On CentOS /usr/local/bin is not included in the PATH when using sudo. 
```

Comment:
I expected to see this on the declaration of the `ARKADE` environment variable, since it may be used multiple times.

Reply:
I moved the comment